### DISCUSSION TITLE
Make active tab in tab bar background the same as the main background?

### DISCUSSION THREAD

**Discussion by kylebarbour** [wezterm/wezterm#4858] [General]
I am using a colorscheme that does not customize the fancy tab bar (Spacemacs). I would like to make the background of the tab the same as the main background, so that there is no obvious border between the tab bar and the main window. Using a minimal `.wezterm.lua` with only the required preamble and `config.color_scheme='Spacemacs (base16)'`, the appearance is:

![image](https://github.com/wez/wezterm/assets/4647176/9781d87c-716d-496e-86fa-d7d595050432)

Note the color difference and resulting border between the active tab background in the tab bar and the main window. What I'm trying to accomplish is the following (edited in GIMP):

![image](https://github.com/wez/wezterm/assets/4647176/4bfd364a-628f-4db4-9845-657b6d93774d)

Note the border being absent and the background of both being identical.

How can this be done? I've reviewed the documentation (particularly [Native (Fancy) Tab Bar appearance](https://wezfurlong.org/wezterm/config/appearance.html#native-fancy-tab-bar-appearance)), but setting the available options doesn't seem to accomplish this.

Thanks!

**Comment by wez** [comment]
Something like this:

```lua
local wezterm = require 'wezterm'
local config = wezterm.config_builder()

-- The color scheme you want to use
local scheme = 'Spacemacs (base16)'
-- Obtain the definition of that color scheme
local scheme_def = wezterm.color.get_builtin_schemes()[scheme]

config.color_scheme = scheme
config.colors = {
  tab_bar = {
    active_tab = {
      bg_color = scheme_def.background,
      fg_color = scheme_def.foreground,
    }
  }
}

return config
```

**Reply by Amparose** [reply]
Wonderful! Where/how can I get a list of colour names available, i.e. all the options available under `scheme_def`?

**Comment by kylebarbour** [comment]
You're a superstar! Thank you so much for the lovely software, and for the swift and helpful reply.
